content/swiftui/concepts/viewmodifier/terms/foregroundColor/foregroundColor.md.md

```diff
@@ -0,0 +1,56 @@
+﻿---
+Title: `.foregroundColor()`
+Description: The `.foregroundColor()  modifier sets the foreground color displayed in a view`.
+Subjects:
+ - 'SwiftUI'
+Tags: 
+ - 'SwiftUI'
+ - 'Modifier'
+ - 'View'
+ CatalogContent:
+ - 'learn-swiftUI'
+ - docs/content/swiftui/concepts/viewmodifier/terms/foregroundColor
+---
+
```

Comment:
You still need to remove the extra line breaks. There shouldn't be more than one line break in a row.

Reply:
Thanks! I have updated the file and committed.